text/0266-move-oteps-to-spec.md

```diff
@@ -0,0 +1,68 @@
+# Move OTEPS to the Specification repository
+
+Let's move OTEP documentation and PRs back into the github.com/open-telemetry/opentelemetry-specification repository.
+
+## Motivation
+
+Moving OTEPs back into the specification solves two main issues:
+
+- Maintaining its tooling infrastructure (currently woefully out of date)
+- Bringing it into the existing triage and voting process currently within the
+  Specification.
+
+## Explanation
+
+Originally, OTEPs were kept as a separate repository to keep disjoint/disruptive designs as a separate repository.  There are a few differences between a normal PR and an OTEP:
```

Comment:
There are a few OTEPs that don't concern the specification, but semantic conventions. Should those go with the specification, and, if need arises, be opened in the specification repository in the future? Or should we have a separate OTEPs in the semantic conventions repo?

Reply:
I believe these move with the specification, e.g. the split of semconv repository, similarly to how proto changes are proposed as OTEPs.  Any semconv proposal that impacts more than semconv should be an OTEP and done in the spec repository.

Anything that ONLY involves semantic conventions should be proposed / dealt with within semconv itself, similar to how language-specific proposals happen in language-specific repositories.  E.g. the proposal around restructuring yaml files or creating the attribute registry -> those are and *were* done only within the semconv repository.